src/commands/branch.rs

```diff
@@ -360,6 +360,13 @@ fn cmd_branch_list(
                 }
             }
             print_branch_target(formatter, Some(remote_target))?;
+            if remote != "git" && branch_target.local_target.is_none() {
+                writeln!(
+                    formatter,
+                    "  (will be *deleted permanently* on the remote on the next `jj git push`. \
+                     Use `jj branch forget` to prevent this)"
+                )?;
+            }
```

Comment:
No need to repeat this message for each remote (because it applies to all remotes)?

Reply:
Good point.

> EDIT: view guarantees that if local_target is None, remote_targets is not empty.

Unfortunately, we mess up this guarantee with the artificially added "git" remote. I don't want to print a message for that, as it would have to say "jj git export". This might be slightly confusing if both a real remote and `@git` are present, but that should be rare and probably not worth printing a message for every remote.